raiden-ts/tests/e2e/provider.ts

```diff
@@ -30,7 +30,7 @@ export class TestProvider extends Web3Provider {
           seed: 'testrpc_provider',
           network_id: 1338,
           db: memdown(),
-          logger: console,
+          // logger: console,
```

Comment:
Why comment instead of removing?

Reply:
I uncomment eventually to debug e2e tests, and some time seems like I forgot to comment out again 😅

Reply:
Can we make it activatable by some environment variable or something? :P

Reply:
Haha, you guys hate logs, this is more for local testing & test debugging in some very specific cases, but if someone feels the need, we can improve it at some point ;)

Reply:
Well not hate. I like them when there is an actual issue. But they come with a performance overhead and they can be overwhelming when they are too verbose. 

In the context ofthis comment though, I would say not as much as I hate commented code :P 

The environment variable here was an alternative to commenting and uncommenting the logger code for the tests.